### ISSUE TITLE
Implement card background color options and test card template refactor for a11y

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#2426]
# Issue

We are moving the events and aggregator template to use the updated card template and this issue will need to be completed so that each event and aggregator item on the Admissions website will render without a fixed white background. 

Add color background options for white, black, gray, and gold for the card component and test them with various background combinations. 
- [x] Adjust LBS block background color options for `white`, `black`, `gray`, and `gold` to appear as options within the card component. 
- [x] Update LBS block background color class names, `white` with `bg--light` might be missing: https://github.com/uiowa/uids/blob/i309-event-card-variant/src/components/background/_background-mixins.scss. 
- [x] Add background.css component to `uids.base.libraries.yml`
- [x] Remove specific background color rules from `card.scss` (https://github.com/uiowa/uiowa/blob/feature_card_option_updates/docroot/themes/custom/uids_base/scss/components/card.scss#L105-L123) and `admin.scss`(https://github.com/uiowa/uiowa/blob/feature_card_option_updates/docroot/themes/custom/uids_base/scss/admin.scss#L278-L289). 
- [x] Find/Replace search for class names listed `background-mixins.scss` in `uids_base` to adjust for new background color names. 
- [x] Update hook to set `bg--white` as background color for existing cards?

## Accessibility testing for v3 sites

Once https://github.com/uiowa/uids/issues/400 is ready, proceed with the testing listed below. 

Now that the card no longer has a standard white background color.   We must test all color contrast variations with the card and related templates listed above to make sure they have accessible color contrast. 

### Gold section background

- [x] Gold section background with Black card background
- [x] Gold section background with white card background
- [x] Gold section background with gold card background
- [x] Gold section background with gray card background

### Black section background

- [x] Black section background with Black card background
- [x] Black section background with white card background
- [x] Black section background with gold card background
- [x] Black section background with gray card background

### Gray section background

- [x] Gray section background with Black card background
- [x] Gray section background with white card background
- [x] Gray section background with gold card background
- [x] Gray section background with gray card background

### White section background

- [x] White section background with Black card background
- [x] White section background with white card background
- [x] White section background with gold card background
- [x] White section background with gray card background

### Brain pattern section background

- [x] Brain pattern section background with Black card background
- [x] Brain pattern section background with White card background
- [x] Brain pattern section background with Gray card background
- [x] Brain pattern section background with Gold card background

### Brain pattern gold section background

- [x] Brain pattern gold section background with Black card background
- [x] Brain pattern gold section background with White card background
- [x] Brain pattern gold section background with Gray card background
- [x] Brain pattern gold section background with Gold card background

### Brain pattern black section background

- [x] Brain pattern black section background with Black card background
- [x] Brain pattern black section background with White card background
- [x] Brain pattern black section background with Gray card background
- [x] Brain pattern black section background with Gold card background

### No card background

- [x] Brain pattern black section background with no set card background
- [x] Brain pattern section background with no set card background
- [x] Brain pattern gold section background with no set card background
- [x] Gold section background with no set card background
- [x] Gray section background with no set card background
- [x] White section background with no set card background
- [x] Black section background with no set card background

## Accessibility testing for v2 sites

We might want to continue to keep a sold white background for cards in `v2`.
